docs/platform/api/using-the-linode-cli.md

```diff
@@ -0,0 +1,314 @@
+
+To delete a resource:
+
+    linode-cli <resource-name> delete <resource-id>
+
+Creating or updating resources follows the same pattern, with additional parameters specific to the resource:
+
+    linode-cli <resource-name> create --color blue
+
+Specific examples of each pattern are covered in the [Examples](#examples) section.
+
+
+## Options
+
+### Help
+
+You can view information about any part of the CLI, including available actions and required parameters, with the `--help` flag:
+
+    linode-cli --help
+    linode-cli linodes --help
+    linode-cli linodes create --help
+
+### JSON Output
+
+The CLI will return output in tabulated format for easy readability. If you would prefer to work with JSON, use the `--json` flag. Adding the `--pretty` flag will format the JSON output to make it more readable:
+
+    linode-cli regions list --json --pretty
+
+{{< highlight json >}}
+[
+  {
+    "country": "us",
+    "id": "us-central"
+  },
+  {
+    "country": "us",
+    "id": "us-west"
+  },
+  {
+    "country": "us",
+    "id": "us-southeast"
+  },
+  {
+    "country": "us",
+    "id": "us-east"
+  },
+  {
+    "country": "uk",
+    "id": "eu-west"
+  },
+  {
+    "country": "sg",
+    "id": "ap-south"
+  },
+  {
+    "country": "de",
+    "id": "eu-central"
+  },
+  {
+    "country": "jp",
+    "id": "ap-northeast"
+  },
+  {
+    "country": "jp",
+    "id": "ap-northeast-1a"
+  }
+]
+{{< /highlight >}}
+
+### Machine Readable Output
+
+You can also display the output as plain text. By default, tabs are used as a delimiter, but you can specify another character with the `--delimiter` option:
+
+    linode-cli regions list --text
+
+{{< highlight text >}}
+id	country
+us-central	us
+us-west	us
+us-southeast	us
+us-east	us
+eu-west	uk
+ap-south	sg
+eu-central	de
+ap-northeast	jp
+ap-northeast-1a	jp
+{{< /highlight >}}
+
+    linode-cli regions list --text --delimiter
+
+{{< highlight text >}}
+id;country
+us-central;us
+us-west;us
+us-southeast;us
+us-east;us
+eu-west;uk
+ap-south;sg
+eu-central;de
+ap-northeast;jp
+ap-northeast-1a;jp
+{{< /highlight >}}
+
+## Examples
+
+Almost any task that can be performed through the Linode Manager or API can also be done with the CLI. This section will review some common examples relating to Linodes, domains, Block Storage Volumes, NodeBalancers, and account details.
+
+{{< caution >}}
+Many of the actions described, such as adding new Linodes, NodeBalancers, or Block Storage Volumes, will result in charges to your account. There is no confirmation prompt when using these commands.
+{{< /caution >}}
+
+### View Available Resources
+
+To view available static resources such as regions, types, images, etc., use the following pattern:
+
+    linode-cli regions list
+    linode-cli types list
+    linode-cli images list
+
+### Linodes
+
+Tasks related to Linode instances are performed with `linode-cli linodes [ACTION]`.
+
+1.  List all of the Linodes on your account:
+
+        linode-cli linodes list
```

Comment:
You could include examples of filtering here.  For example, `linode-cli linodes list --region us-east` to see all Linodes in one region.  The `--help` tells you what columns are filterable.